charts/pega/charts/hazelcast/templates/_helpers.tpl

```diff
@@ -0,0 +1,38 @@
+{{- define "hazelcastName" -}} pega-hazelcast {{- end -}}
+{{- define "hazelcastEnvironmentConfig" -}} pega-hz-env-config {{- end -}}
+
+
+{{- define "isHazelcastEnabled" }}
+ {{- if .Values.enabled -}}
+  true
+ {{- else -}}
+  false
+ {{- end -}}
+{{- end }}
+
+{{- define "generatedHazelcastServicePodAnnotations" }}
```

Comment:
Looks like this definition will be overridden in helm deployments other than CMC. Can we add a comment to be clear on how this definition will be used in similar lines to [pega/_helpers.tpl](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/templates/_helpers.tpl#L226)

Reply:
good point. I will add a comment.